js/src/jit/CompactBuffer.h

```diff
@@ -216,6 +216,11 @@ class CompactBufferWriter {
       writeByte((ptrWord >> (i * 8)) & 0xFF);
     }
   }
+  void writeBytes(const uint8_t* data, size_t len) {
```

Comment:
Is this being used somewhere?

Reply:
Yep, exactly, it's used in the ctor for `CacheIRWriter` from an AOT stub.